### ISSUE TITLE
macOS 14.2 beta 4 bug: brightness slider not working

### ISSUE THREAD

**Issue by chadmundoholding** [MonitorControl/MonitorControl#1474]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

The brightness change functions, however the bar will not visually slide, upon the newest update. It would be hard to show by screenshots. Essentially, if I try sliding the brightness, the brightness immediately changes, but the new, correct placement is only visible upon hiding, then re-showing MonitorControl. The sound slider bar works just as before. When I setup the percents, those change right away, but the slider not so much. It will occasionally move, if I have more settings on.

### Steps to reproduce

1. Click on MonitorControl in the Menu Bar
2. Click or drag the brightness slider bar to increase or decrease the brightness
3. If the position didn't change, but the brightness did, then click on MonitorControl to hide it
4. Click again into MonitorControl, and see that it has now moved

### Expected behavior

The slider should react immediately and move when I click it.

### Anything else?

![Screenshot 2023-12-01 at 1 02 37 AM](https://github.com/MonitorControl/MonitorControl/assets/152590955/b7d2aec3-481f-442f-b320-491c765b6010)
Above screenshot shows, I clicked to lower the brightness, the percent agrees with what I am seeing, however the slider is still well over 50%

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
This is a general macOS 14.2 beta 4 bug. Other sliders are broken system-wide (take a look at the volume slider of the Music app for example).

**Comment by waydabber** [comment]
Fixed with 14.2 RC
